packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -85,6 +85,7 @@ export default deepFreeze(
       tracking: {
         gaID: null,
         trackErrors: true,
+        customWebAnalytics: null,
```

Comment:
If I understand the question correct, my answer is yes it will look like

```
customWebAnalytics: function (config?) {
         return {
           init: () => { /* initialization actions */ },
           trackPageView: (pathname, search) => {},
           trackError: (description) => {},
           trackEvent: (category, action, labelOrValue, value) => {},
           context: null,
           isEnabled: () => false,
         }
       }
```

Reply:
I meant whether we have a type definition for Config somewhere. Maybe not.